website/docs/cloud-docs/integrations/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+page_title: Integrations - HCP Terraform
+description: >-
+  This overview page lists HCP Terraform integrations with various third-party platforms and systems. 
+---
+
+# Overview
+
+The HCP Terraform ecosystem features a variety of integrations – applications
+designed to extend Terraform’s capabilities across various third-party systems
+and platforms. As demand grows for managing complex infrastructure across
+multiple clouds, data centers, and third-party services, our integrations are
+expanding.
+
+This section lists the current HCP Terraform integrations, which are API-based
+and leverage HCP Terraform's robust set of APIs. Each integration has its own
+dedicated page where you can find more details:
+
```

Comment:
Suggesting making these all `-` because this isn't a list you need to do in order.